### PR TITLE
Implements Init_One_Time_Systems and Init_Bulk_Data.

### PR DESCRIPTION
**Pull request by OmniBlade**

### CODE REVIEW

File: src/game/engine/init.cpp [hunk 1]
```diff
@@ -269,6 +269,67 @@ void Reinit_Secondary_Mixfiles()
...
+    ini.Load(fc);
+
+    // TODO size is ARRAY_SIZE(g_TutorialText), needs adjustments to hooked global or standalone.
+    for (int i = 0; i < 225; ++i) {
+        char entry[20];
+        char buff[128];
+        g_TutorialText[i] = nullptr;
+        sprintf(entry, "%d", i);
+
+        if (ini.Get_String("Tutorial", entry, "", buff, sizeof(buff)) != 0) {
```
Comment (ghost) [inline src/game/engine/init.cpp]: Best to use "> 0" here as we do for other uses of Get_String.

File: src/game/engine/init.cpp [hunk 2]
```diff
@@ -269,6 +269,67 @@ void Reinit_Secondary_Mixfiles()
...
+    TemplateTypeClass::One_Time();
+    OverlayTypeClass::One_Time();
+    SmudgeTypeClass::One_Time();
+    TerrainTypeClass::One_Time();
+    UnitTypeClass::One_Time();
+    VesselTypeClass::One_Time();
+    InfantryTypeClass::One_Time();
+    AnimTypeClass::One_Time();
+    AircraftTypeClass::One_Time();
+    HouseTypeClass::One_Time();
```
Comment (tomsons26) [inline src/game/engine/init.cpp]: Should be HouseClass::One_Time

File: src/game/engine/init.cpp [hunk 3]
```diff
@@ -480,7 +541,7 @@ void Init_Color_Remaps()
  */
 void Init_Mouse()
 {
-   if (!g_MouseInstalled) {
+    if (!g_MouseInstalled) {
```
Comment (tomsons26) [inline src/game/engine/init.cpp:545]: intended formatting change?

**Comment by tomsons26** [comment]
i think Init_Heaps should be added in this too, we should have the heaps, if not those missing should be added. Speech stuff in it is preferable to be in its own function this calls as its technically nothing to do with heaps.

File: src/game/engine/init.cpp (hunk 3 above)
Comment (OmniBlade) [reply src/game/engine/init.cpp:545]: Lets just go with yes and move on ;)

**Review by ghost — APPROVED** [review]

**Pull request merged**